src/engraving/dom/fret.h

```diff
@@ -291,6 +296,8 @@ class FretDiagram final : public EngravingItem
 
     bool m_showFingering = false;
     std::vector<int> m_fingering = std::vector<int>(m_strings, 0);
+
+    static std::map<String, String> s_harmonyToDiagramMap;
```

Comment:
It can be declared directly in the cpp file. Also, std::unordered_map is the preferred type for this task